account_move_name_sequence/tests/test_account_move_name_seq.py

```diff
@@ -164,3 +178,379 @@ def test_in_refund(self):
         in_refund_invoice.button_draft()
         in_refund_invoice.action_post()
         self.assertEqual(in_refund_invoice.name, move_name)
+
+
+class ThreadRaiseJoin(threading.Thread):
+    """Custom Thread Class to raise the exception to main thread in the join"""
+
+    def run(self, *args, **kwargs):
+        self.exc = None
+        try:
+            return super().run(*args, **kwargs)
+        except BaseException as e:
+            self.exc = e
+
+    def join(self, *args, **kwargs):
+        super().join(*args, **kwargs)
+        # raise exception in the join
+        # to raise it in the main thread
+        if self.exc:
+            raise self.exc
+
+
+@tagged("post_install", "-at_install", "test_move_sequence")
+class TestSequenceConcurrency(TransactionCase):
```

Comment:
Could you create a new file for these kind of unittest, please?

What about `account_move_name_sequence/tests/test_sequence_concurrency.py`?